### ISSUE TITLE
Node.js Collection Publication Week of March 6 and March 13

### ISSUE THREAD

**Issue by ZibbyKeaton** [nodejs/evangelism#284]
Hey All - I received another submission. It is really technical and a bit over my head to decide whether or not it would be a good add to Node.js Collection. I would imagine it would fall under the "How to" section. I did find some grammatical issues that we would need to edit if we did want to go through with this. Please let me know what you think by EOD Friday (March 10): https://medium.com/@jacopodaeli/f7ab3d9e06fc#.b29qobs84

**Comment by davidfekke** [comment]
It is a good introduction on which keyword to use when declaring a variable in JS, and he compares `const` to the one that is used in C and C++.

**Comment by ZibbyKeaton** [comment]
Do you think this is valuable and relevant to the Node.js community @davidfekke?

**Comment by anaptfox** [comment]
Hi @ZibbyKeaton, besides some formatting and grammar, I think overall the content the content is good.

There are a couple of places where he could have elaborated more like when he talked about "no rebinding will happen." There could have been an explanation on why the second example was different from the first. 

Also, a mention of Immutable.js could be helpful when talking about "How to make a value immutable."

**Comment by davidfekke** [comment]
I agree with Taron. This does have value.

**Comment by ZibbyKeaton** [comment]
@davidfekke as an aside did you receive my request to publish your Medium post. I'm not seeing it on the publication, so I want to make sure the request went through.

**Comment by davidfekke** [comment]
I did not. What do I need to do?

**Comment by ZibbyKeaton** [comment]
@davidfekke please submit it to be added to the publication. I've been having some issues here. It seems to work when folks submit it directly.

**Comment by davidfekke** [comment]
When I click on the "Add to publication" link, it just has a checkbox with
the following text: "Allow publications to email you and ask to include
your story".

I do not have an option to add it to the-nodejs-collection.

**Comment by ZibbyKeaton** [comment]
@davidfekke please click the box and then I can add it in - I will get an email prompt to do this. I'll also add you as an editor to the publication.

**Comment by davidfekke** [comment]
I think I have it checked.

**Comment by davidfekke** [comment]
I just added it.

Thanks Zibby.

**Comment by ZibbyKeaton** [comment]
I just posted this from the Node.js Foundation Medium page. Let me know if you think it would be good for the Node.js Collection: https://medium.com/@nodejs/node-js-and-apis-a-match-made-in-scaling-heaven-9ffb0b1d3c9a#.rn51ofnai

**Comment by vdeturckheim** [comment]
@ZibbyKeaton LGTM

**Comment by ZibbyKeaton** [comment]
In addition, if you want to help spread the word about Netflix using Node.js. Below are a few social posts that you can use :) Feel free to create your own as well:

A post about how Netflix is using @nodejs and general best practices when it comes to API development with Node.js: http://bit.ly/2mpBZ7R
Click-to-tweet: http://ctt.ec/Zl360

Node.js and APIs: A Match Made in Scaling Heaven: http://bit.ly/2mpBZ7R
Click-to-tweet: http://ctt.ec/M3bkV

**Comment by ZibbyKeaton** [comment]
New request on submitting an article to Node.js Collection. It focuses on JavaScript's number type in details. It's a pretty hefty article and I am not aware of this topic area. Could this group provide their thoughts and feedback on this by Friday (March 17): https://medium.com/@maximus.koretskyi/javascripts-number-type-8d59199db1b6#.b9kvk4wsf. Let me know if its something you think the Node.js community would be interested in.

**Comment by davidfekke** [comment]
This is a great post. One of the things that drives me nuts about JavaScript is the way it handles math. He explains why 0.1+0.2 does not equal 0.3 in JavaScript.

He goes into great detail explaining how floating point numbers are stored in binary.

**Comment by Trott** [comment]
>  One of the things that drives me nuts about JavaScript is the way it handles math. He explains why 0.1+0.2 does not equal 0.3 in JavaScript.

FWIW, that peculiarity exists in most programming languages including C, C++, Java, Python, Ruby... http://0.30000000000000004.com/

**Comment by ZibbyKeaton** [comment]
@Trott I saw a comment earlier from you via email, but I can't seem to find it here. It sounded like your one critique or suggest for him was that he should mention that this idea of 0.1+0.2 not equally 0.3 in JavaScript is something that is common in most programming languages. Does this completely change the context of the post, i.e. can you apply what he is saying here to all programming languages or do you think he should simply make the reference? I want to make sure I give him concrete and correct feedback here. Thanks in advance!

**Comment by davidfekke** [comment]
He does reference that in his post. I don't think he needs to change anything in his post.

**Comment by Trott** [comment]
>  Does this completely change the context of the post, i.e. can you apply what he is saying here to all programming languages or do you think he should simply make the reference?

They make it clear in the post that that particular issue is not at all unique to JavaScript, which is why I deleted my comment. Ignore me. :-D

**Comment by Trott** [comment]
(If it hasn't happened already, given the subject, someone might want to go through the text keeping an eye out for things that veer away from helpful and towards newbie-shaming. Hopefully there isn't anything like that, but always a good idea to double-check when someone refers to "novice" developers as this post does.)

**Comment by ZibbyKeaton** [comment]
Good point. @Trott if you see anything that jumps out, please share in this thread. I'll do a read through, but will also look to this team to help me as I might know all the nuances of what a newbie might find discouraging.

**Comment by ZibbyKeaton** [comment]
Hey All - I have a post for folks to review, this is based off of the [original post](https://medium.com/@jacopodaeli/f7ab3d9e06fc#.b29qobs84) that Jacopo sent us. He made some formatting and grammar edits and I did some edits of my own (suggestions only). Could folks add in their edits either directly in the document or in this thread by next Wednesday (March 22). I want to make sure that I'm not missing anything here and would like technical eyes on it to make sure my changes do not hurt to integrity of the technical concepts. https://docs.google.com/document/d/1aS1iXHTsCHUFt7MYH2vcQHBF0iOtdPZ5FGXF6PN9fis/edit#

**Comment by vdeturckheim** [comment]
@ZibbyKeaton lgtm

**Comment by ZibbyKeaton** [comment]
Hey All - I made some comments on this Medium page based on this groups feedback: https://medium.com/@maximus.koretskyi/javascripts-number-type-8d59199db1b6#.pw8dkgk1a. A few of you are editors of our Medium publication, so please go in there and make suggestions and edits as well. If you are not an editor and want to become one, please let me know - I can easily add you. Please have all your edits into this one by EOD Wednesday next week as well - that would be March 22 :)

**Comment by ZibbyKeaton** [comment]
Another post for this group's approval: https://medium.com/@vdeturckheim/meet-the-people-who-make-node-js-colin-ihrig-4441d283d128#.sl7d73apm. This is an with CTC member Colin Ihrig. I have some small edits to make to this, but before I go through with them, please let me know if this group would like to add it to our publication by EOD Wednesday too. Lots of due dates for Wednesday :)

**Comment by vdeturckheim** [comment]
I will not comment my own article ;) 

@**ALL** Do you think such interviews are a good idea for the publication overall?

**Comment by anaptfox** [comment]
That was an awesome read!! 

I think interviews are great. It's always helpful for beginners to see the path that others took.

**Comment by davidfekke** [comment]
I loved it!

**Comment by bnb** [comment]
@vdeturckheim Love it as well - great job.

**Comment by Trott** [comment]
Minor typo in the (totally awesome) interview: `technical of community related` -> `technical or community related`

(Is this the right place to leave a comment like this?)

**Comment by vdeturckheim** [comment]
@Trott thanks a lot. Just fixed it!

**Comment by ZibbyKeaton** [comment]
@vdeturckheim I also saw that you used adminstrivia in one of the sections. Don't know if this is a new word or if you meant to stay administrative. I can never keep up with new lingo. We'll give folks until Friday to make any last minute edits or comments here. Also, please let me know if anyone has changes that they want to make to this article by 5pm ET today: https://medium.com/@maximus.koretskyi/javascripts-number-type-8d59199db1b6#.pw8dkgk1a

**Comment by ZibbyKeaton** [comment]
We also had the following come in as a contributed post for the series. I think that this should work as long as they keep it educational and not promotional (which they emphasize). Can this group tell me if they agree with this by Friday and I'll request a draft for our review: 

**Request**
_we are about to write an article about a nodejs project we developed for a customer and I would like it to be considered for publication in the Node.js medium collection.

We already informed our customer about our intention to write about the product, and they are fine with that.

The project is a server application which allows to create and manage digital signature workflows, while using our customer's pre existing secure signature infrastracture (http://www.validsign.com),  

This won't contain advertisement of the companies involved or the product, it will be a striclty technical article but the product name and our companies name might be mentioned: could that be a guideline violation?

The article will be structured as a list of "lessons learned": which challenges we faced, and how we solved them using node an other tools such as typescript and docker.

If you are interested I will provide a draft as soon as it is ready (a couple of working days)._
